test/e2e/operator.go

```diff
@@ -70,3 +72,42 @@ func failTestAndPrintLogs(t *testing.T, namespace string, err error) {
 
 	t.Fatal(err)
 }
+
+func cleanup(t *testing.T, ctx *framework.TestCtx) {
```

Comment:
this kind of breaks the single responsibility principle, since the `cleanup` function also gets and prints k8s events

I don't think it should be in one function

Reply:
the name of the function should be something that implies there will be some printing there, like `cleanupAndShowLogs` (not the best name ever, I know)

I see 3 new functions here:
- `printOperatorLogs`
- `getKuberneresEvents`
- `printKubernetesEvents`

Reply:
also `getOperatorPod` and `getOperatorLogs` signatures could benefit from refactoring 
and `failTestAndPrintLogs` could reuse `printOperatorLogs`